### DISCUSSION TITLE
Help Setting up

### DISCUSSION THREAD

**Discussion by MULTIPL3X** [MartinStoffel/tigo#10] [Q&A]
Hello,

i got an issue setting up this integration. Installation via hacs and Login in afterwards seem to work fine but then it does not show me any devices or entities. Did i miss a step while setting up? For example put in some data in a config file or somthing similar. 

![Hassio Tigo Screenshot](https://github.com/user-attachments/assets/2b6cb00e-a6d5-44cd-aed6-e74c9ec5d22a)

Ps: I also have the premium subscription for tigo also already played a little bit here "https://ei.tigoenergy.com/api-docs/index" so the api seems to work.

**Comment by MartinStoffel** [comment]
Hi @MULTIPL3X 

There is no manual configuration needed. Try to restart your HA. If this does not help, double check the System ID you entered.

**Comment by MULTIPL3X** [comment]
Hi @MartinStoffel,

thank you for your fast reply. i already checked and restartet a few times when i opened the discussion. Just did it one more time to be sure and still not working.

**Reply by MartinStoffel** [reply]
hmm... I don't have an idea what else might be wrong. 

(By the way, this is integration is reverse-engineered fom the webpage and does not need the paied api.)

**Reply by MULTIPL3X** [reply]
Just an idea, is it possible that there is a problem with the latest version of HA running on an Raspberry Pi?
because this is the environment i am using at the moment. I might be able to test it on a different environment.

![Hassio version](https://github.com/user-attachments/assets/f64511e3-c152-414b-9bed-ffa8ffc840a4)

**Reply by MartinStoffel** [reply]
I don't think so. Did you check the logs?  I might add some additional debug logging code if you want.

May I ask you how big your system is? And well, sometimes waiting a day helps - Tigo now and then has also some issues...
Or I could offer to debug on my side, if you are willing to share the needed info.
